### PR TITLE
Edit todo item component

### LINKED ISSUE #7: Edit a Todo Item
Addressing the following user story:
As a user, I want to have a TODO list where I can add, delete, edit and check off items so I can stay productive during my day

Wireframe:
image

### PR DESCRIPTION
**Pull request by emmayjiang**
Closes #7. Creates a new component to edit a todo item. Leaving as a draft PR as it is currently not hooked up to the detailed todo item view.

### CODE REVIEW

**Comment by emmayjiang** [comment]
image

File: EXP-Client/progress/TaskItem.js [hunk 1]
```diff
@@ -2,13 +2,12 @@ import React, { useState } from 'react';
...
-export default function TaskItem({ task }) {
-  const [isEditing, setIsEditing] = useState(false);
-  const [name, setName] = useState(task.name);
-  const [deadline, setDeadline] = useState(task.deadline);
-  const [isCompleted, setIsCompleted] = useState(task.completed);
-  const [rating, setRating] = useState(task.rating);
+export default function TaskItem({ task, setViewingTask }) {
+  const [isEditing, setIsEditing] = React.useState(false);
+
+  console.log("viewing task: " + task.name, task.deadline, task.priority, task.completed);
```
Comment (AlexanderChiuy) [inline EXP-Client/progress/TaskItem.js]: looks like debugging code ?

File: EXP-Client/todo/TaskList.js [hunk 2]
```diff
@@ -36,13 +36,18 @@ export default function TaskList({ tasks, setIsPanelActive }) {
...
-            <View style={styles.text}>
-              <Text
-                multiline
-                style={{ fontSize: 20 }}  
-              >{name}</Text>
-              <Text style={{ fontSize: 10 }}>{deadline}</Text>
-            </View>
+            <Pressable style={styles.pressable} onPress={() => {
+              setViewingTask({ name, deadline, priority, completed });
+              console.log("viewing task: " + name);
```
Comment (AlexanderChiuy) [inline EXP-Client/todo/TaskList.js]: Same for here

**Review by AlexanderChiuy — CHANGES_REQUESTED** [review]

**Review by AlexanderChiuy — APPROVED** [review]
LGTM

**Pull request merged**